Jenkinsfile

```diff
@@ -83,37 +84,47 @@ pipeline {
             steps {
                 script {
                     sh """
-                    docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd repo && eups declare -r . -t saluser && setup ts_MTDome -t saluser && export LSST_DDS_IP=192.168.0.1 && pytest --color=no -ra --junitxml=tests/results/results.xml\"
+                    docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd repo && pip install --ignore-installed -e . && eups declare -r . -t saluser && setup ts_MTDome -t saluser && pytest --junitxml=\${XML_REPORT}\"
                     """
                 }
             }
         }
     }
     post {
         always {
-            // The path of xml needed by JUnit is relative to
-            // the workspace.
-            junit 'tests/results/results.xml'
+            // Publish the HTML report
+            publishHTML (target: [
+                allowMissing: false,
+                alwaysLinkToLastBuild: false,
+                keepAll: true,
+                reportDir: 'jenkinsReport/',
+                reportFiles: 'index.html',
+                reportName: "Coverage Report"
+              ])
+
             sh "docker exec -u saluser \${container_name} sh -c \"" +
                 "source ~/.setup.sh && " +
                 "cd /home/saluser/repo/ && " +
-                "setup ts_MTDome -t saluser && " +
+                "setup ts_salobj -t saluser && " +
```

Comment:
I'm a bit confused here. Don't you need to be in the ts_MTDome directory and have it setup before you build its docs? Yet you are setting up ts_salobj, not ts_MTMount.

Reply:
What happens if `package-docs build` fails? I hope it doesn't go on and try to upload the docs. Naively I would have expected this to be a script, and call unstable if the result is not 0. I see ts_salobj does things the same way. I wonder if that means it needs updating as well?

Reply:
Thanks for catching that. I basically copied that part from the ts_salobj Jenkinsfile and forgot to modify that line, which I will do now.

Reply:
Also, this id done from the `repo` directory and not `repos`.